lib/widgets/editor/tree_widgets/path_tree.dart

```diff
@@ -34,6 +34,7 @@ class PathTree extends StatefulWidget {
   final num? pathRuntime;
   final bool holonomicMode;
   final PathConstraints defaultConstraints;
+  final num? previousPathRuntime;
 
```

Comment:
Perhaps instead of taking in both `pathRuntime` and `previousPathRuntime` now it could just take in a nullable runtime display widget? Then all this widget has to do is display that runtime widget if its provided. Then it shouldn't need the `didUpdateWidget` stuff either. Same thing goes for the auto tree.

Reply:
Good idea, made it so split_path_editor has the RuntimeDisplay object and that is used for the PathTree instead.